actions/installer.yml

```diff
@@ -0,0 +1,27 @@
+name: slsa-verifier-installer
+description: 'Installs SLSA verifier and adds it to your PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  verifier-release:
+    description: 'slsa-verifier release version to install'
+    required: false
+    default: 'v1.2.0'
+  install-dir:
```

Comment:
Do we need this option? What's the use case?

Reply:
Done